notebooks/itslive/itslive.ipynb

```diff
@@ -0,0 +1,234 @@
+    "urls = []\n",
+    "params = ui.build_params()\n",
+    "if params is not None:\n",
+    "    urls = ui.get_granule_urls(params)\n",
+    "    # Print the first 10 granule URLs\n",
+    "    for url in urls[0:10]:\n",
+    "        print(url)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Filtering granules by year and month"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#4: This will query AWS(where the granules are stored) so we know the total size of our first N granules\n",
+    "# This may take some time, try reducing the selected area or constraining the other parameters to download a reasonable number of granules.\n",
+    "url_list = [url['url'] for url in urls]\n",
+    "# urls\n",
+    "filtered_urls = ui.filter_urls(url_list, max_files_per_year=5, months=[3,4,5,6,7])\n",
+    "\n",
+    "# max_granules = 100\n",
+    "# sizes = ui.calculate_file_sizes(filtered_urls, max_granules)\n",
+    "# total_zise = round(sum(sizes)/1024,2)\n",
+    "# print(f'Approx size to download for the first {max_granules:,} granules: {total_zise} MB')\n",
+    "print(len(filtered_urls))\n",
+    "filtered_urls"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Downloading the data\n",
+    "**Now that we have our list of data granules we can download them from AWS.**\n",
+    "\n",
+    "If this notebook is running inside AWS we could load the granules into a Dask cluster and reduce our processing times and costs.\n",
+    "Let's get some coffee, some data requests are in the Gigabytes realm and may take a little while to be processed. \n",
+    "Once that your status URL says is completed we can grab the HDF5 data file using the URL on the same response!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#5 This will download the first 50 velocity pairs\n",
+    "files = ui.download_velocity_pairs(filtered_urls, start=0, end=50)\n",
+    "files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Working with the data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import glob\n",
+    "import xarray as xr\n",
+    "import pyproj\n",
+    "import warnings\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "from datetime import datetime\n",
+    "from shapely.geometry import Polygon\n",
+    "warnings.filterwarnings('ignore')\n",
+    "centroid = ui.dc.last_draw['geometry']['coordinates']\n",
+    "# coord = [-49.59321, 69.210579]\n",
+    "#loads an array of xarray datasets from the nc files\n",
+    "velocity_pairs = ui.load_velocity_pairs('data')\n",
```

Comment:
Is 'data' supposed to be a single file path? I tried this on a few of my output files and my velocities were empty. Not sure what I shuold be doing here.

Reply:
data is the name of the directory you want to load the velocity granules from. Clarified in a comment.